src/mongo.js

```diff
@@ -60,6 +60,19 @@ function extend(Mongo: Object) {
       });
     },
 
+    getIndexes() {
+      return new Promise((resolve, reject) => {
+        this.rawCollection().indexes((error, indexes) => {
+          if (error) {
+            reject(indexes);
```

Comment:
```suggestion
            reject(error);
```
🙃

Reply:
I'm so going to abuse this

Reply:
Wow that was careless of me. I don't like this new feature, though; it feels so aggressive.

Reply:
Yeah I agree

Reply:
On your feelings towards the new feature, that is